distribution/docker/build.gradle

```diff
@@ -240,9 +240,17 @@ void addBuildDockerContextTask(Architecture architecture, DockerBase base) {
       }
 
       if (base == DockerBase.CLOUD) {
+        // If we're performing a release build, but `build.id` hasn't been set, we can
```

Comment:
Question, so do we actually build the docker images multiple times?

Reply:
Yes - we build the default image, the UBI image, the Iron Bank context, the Cloud Image and the Cloud ESS image. Apart from UBI and Iron Bank, we also build them on ARM.